test/mocha/10-totp.js

```diff
@@ -245,17 +245,36 @@ describe('TOTP API', () => {
 
 describe('TOTP Database Tests', () => {
   describe('Indexes', async () => {
+    let accountId;
+    let actor;
+    let secret;
     // NOTE: the accounts collection is getting erased before each test
     // this allows for the creation of tokens using the same account info
     beforeEach(async () => {
       await helpers.prepareDatabase(mockData);
+      accountId = mockData.accounts['alpha@example.com'].account.id;
+      actor = await brAccount.getCapabilities({id: accountId});
+      const accountId2 = mockData.accounts['beta@example.com'].account.id;
+      const actor2 = await brAccount.getCapabilities({id: accountId2});
+
+      // creates tokens
+      ({secret} = await brAuthnToken.set({
+        account: accountId,
+        actor,
+        type: 'totp'
+      }));
+      await brAuthnToken.set({
```

Comment:
Perhaps add a note that explains why a second account is being added so a future maintainer doesn't try to simplify this by removing it.